src/utils/StorageHelper.ts

```diff
@@ -33,6 +34,18 @@ class StorageHelper {
         localStorage.setItem(AUTH_KEY, authKey)
         sessionStorage.setItem(AUTH_KEY, '')
     }
+
+    setSiderCollapsedStateInLocalStorage(siderCollapsed: boolean) {
+        localStorage.setItem(
+            SIDER_COLLAPSED_STATE,
+            JSON.stringify(siderCollapsed)
```

Comment:
Do we need to store this as JSON? Can we not do this instead?
```
 localStorage.setItem(SIDER_COLLAPSED_STATE, !!siderCollapsed)
```

Same with `getSiderCollapsedStateFromLocalStorage`, can we do 
```
return !!localStorage.getItem(SIDER_COLLAPSED_STATE)
```

Reply:
Just tried it, but could not make to work, let me know if I'm wrong. Still learning TS.

**Explanation:**

`setSiderCollapsedStateInLocalStorage:`
```js
// Throws: Argument of type 'boolean' is not assignable to parameter of type 'string'.ts(2345) -> setItem requires string
 localStorage.setItem(SIDER_COLLAPSED_STATE, !!siderCollapsed)
// Casting to string and removing the !!
 localStorage.setItem(SIDER_COLLAPSED_STATE, String(siderCollapsed))
```

`getSiderCollapsedStateFromLocalStorage:`
```js
// Receives string  a.e !!"false", evaluates to true
return !!localStorage.getItem(SIDER_COLLAPSED_STATE) 
// An option would be
return storageValue === 'true' ? true : false
```

I followed the recommendations on SO for the implementation.

See: 
* https://stackoverflow.com/a/30644349/3313177
* https://stackoverflow.com/a/2010948/3313177

Let me know if I should update the PR to remove the JSON.parse and replace it with the version above :)

Reply:
Ah you're right. local storage can only store string values.